docs/conf.py

```diff
@@ -0,0 +1,63 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+from sphinx.ext.autodoc import between
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'discord-ext-ipc'
+copyright = '2021, Ext-Creators'
+author = 'Ext-Creators'
+
+# The full version, including alpha/beta/rc tags
+release = 'v2.0.0a'
```

Comment:
Pull the version string from [`__init__.py`](https://github.com/Ext-Creators/discord-ext-ipc/blob/master/discord/ext/ipc/__init__.py#L20) not unlike [`setup.py`](https://github.com/Ext-Creators/discord-ext-ipc/blob/master/setup.py#L8-L11).